pkg/infrastructure/gcp/clusterapi/clusterapi.go

```diff
@@ -209,6 +209,11 @@ func (p Provider) InfraReady(ctx context.Context, in clusterapi.InfraReadyInput)
 		return fmt.Errorf("failed to add firewall rules: %w", err)
 	}
 
+	// Remove the overly permissive firewall rules created by CAPG
```

Comment:
nit: maybe a bit of information about why this is necessary

Reply:
Added more info.